README

```diff
@@ -359,7 +359,7 @@ WARNINGS and TAINT FLAGS:
         will break if /usr is on a separate late-mounted partition, many of its
         dependencies very likely will break sooner or later in one form or
         another. For example, udev rules tend to refer to binaries in /usr,
-        binaries that link to libraries in /usr or binaries that refer to data
+        binaries that link to libraries in /usr, or binaries that refer to data
```

Comment:
if you touch this, please also suffix /usr with /. It's customary in our documentation to suffix paths referring to directories with a /.